lib/rucio/core/config.py

```diff
@@ -34,24 +40,24 @@
 SECTIONS_CACHE_KEY = 'sections'
 
 
-def _has_section_cache_key(section: str) -> str:
-    return 'has_section_%s' % section
+def _has_section_cache_key(section: str) -> HasKey:
+    return HasKey('has_section_%s' % section)
 
 
-def _options_cache_key(section: str) -> str:
-    return 'options_%s' % section
+def _options_cache_key(section: str) -> OptionsKey:
+    return OptionsKey('options_%s' % section)
 
 
-def _has_option_cache_key(section: str, option: str) -> str:
-    return 'has_option_%s_%s' % (section, option)
+def _has_option_cache_key(section: str, option: str) -> HasKey:
+    return HasKey('has_option_%s_%s' % (section, option))
 
 
-def _items_cache_key(section: str) -> str:
-    return 'items_%s' % section
+def _items_cache_key(section: str) -> ItemsKey:
+    return ItemsKey('items_%s' % section)
 
 
-def _value_cache_key(section: str, option: str) -> str:
-    return 'get_%s_%s' % (section, option)
+def _value_cache_key(section: str, option: str) -> ValueKey:
+    return ValueKey('get_%s_%s' % (section, option))
```

Comment:
This is being refactored in this previous open PR: https://github.com/rucio/rucio/pull/7015/files#diff-3c88400ee4ff0b9fca1b3a4f6dd3f58e23b10991a3a4032cbdbcd7028ae5c264